src/gui/pointcloud/qgspointcloudclassifiedrendererwidget.cpp

```diff
@@ -126,6 +127,14 @@ QVariant QgsPointCloudClassifiedRendererModel::data( const QModelIndex &index, i
         }
         case 2:
           return category.label();
+        case 3:
+          const float value = mPercentages.value( category.value() );
+          QString str;
+          if ( value < 0.06 && value != 0 )
+            str = QStringLiteral( "< " ) + QLocale().toString( 0.1, 'f', 1 );
+          else
+            str = QLocale().toString( mPercentages.value( category.value() ), 'f', 1 );
+          return str;
```

Comment:
If I understand correctly, for classes where we don't have stats (when mPercentages does not have entry), we would be still showing zero, which is probably a bit misleading... how about we either leave there as an empty string, or we use a text e.g. "N/A" or "unknown" ?

Reply:
That's a valid concern, looking into it

Reply:
great, thanks!